### PR TITLE
feat: Integarting backend app

### LINKED ISSUE #3: Add Backend app (monorepo?)

### PR DESCRIPTION
**Pull request by tanaytoshniwal**

### CODE REVIEW

File: backend/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+node_modules
+node_modules/
+db/users.db
```
Comment (llsamplell) [inline backend/.gitignore:3]: why this?

Comment (tanaytoshniwal) [reply backend/.gitignore:3]: This represents sample user's database
can use the production database here^^

Comment (llsamplell) [reply backend/.gitignore:3]: Oh cools
Approving!!!

**Review by llsamplell — APPROVED** [review]
LGTM

**Pull request merged**